<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT" verbose="0">
  <test thread-count="5" name="Order(failed)" verbose="0">
    <classes>
      <class name="VenkataRamanapandi.Tests.SubmitOrder">
        <methods>
          <include name="close"/>
          <include name="SubmitMethod"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- VenkataRamanapandi.Tests.SubmitOrder -->
    </classes>
  </test> <!-- Order(failed) -->
  <test thread-count="5" name="Error(failed)" verbose="0">
    <classes>
      <class name="VenkataRamanapandi.Tests.ErrorHandling">
        <methods>
          <include name="close"/>
          <include name="LoginError"/>
          <include name="launchApplication"/>
        </methods>
      </class> <!-- VenkataRamanapandi.Tests.ErrorHandling -->
    </classes>
  </test> <!-- Error(failed) -->
</suite> <!-- Failed suite [Suite] -->
